dev/build_google_component_image.sh

```diff
@@ -173,14 +173,16 @@ function create_component_prototype_disk() {
       --command="sudo DEBIAN_FRONTEND=noninteractive apt-get -y dist-upgrade && sudo apt-get autoremove -y"
   fi
 
-  echo "`date`: Deleting '$BUILD_INSTANCE' but keeping disk"
+  echo "`date`: Setting auto-delete behavior '$BUILD_INSTANCE' disk to false"
   gcloud compute instances set-disk-auto-delete $BUILD_INSTANCE \
     --project $BUILD_PROJECT \
     --account $ACCOUNT \
     --zone $ZONE \
     --no-auto-delete \
     --disk $BUILD_INSTANCE
 
+  sleep 20
```

Comment:
I'm hesitant to poll because if it's a consistency problem it won't guarantee the flag is set. Also, given that the builds take ~15 minutes now this is a negligible increase in wait time.